api/src/main/java/org/ccci/idm/user/Dn.java

```diff
@@ -16,13 +16,11 @@
 public class Dn implements Serializable {
     private static final long serialVersionUID = 5510344429904560934L;
 
+    public static final Dn ROOT = new Dn();
+
     @Nonnull
     private final List<Component> components;
 
-    public Dn() {
-        components = ImmutableList.of();
-    }
-
```

Comment:
ease of usage, that way you don't have to constantly write boilerplate code to create a List or convert a List to an Array when constructing a `Dn` object via the constructor.

Although, you probably shouldn't be constructing a `Dn` object directly anyways, use the `DnUtils.toDn()` method.